src/app/clusters/ota-requestor/DefaultOTARequestorDriver.cpp

```diff
@@ -121,15 +121,13 @@ bool DefaultOTARequestorDriver::ProviderLocationsEqual(const ProviderLocationTyp
     }
 }
 
-void DefaultOTARequestorDriver::HandleError(UpdateFailureState state, CHIP_ERROR error) {}
-
 void DefaultOTARequestorDriver::HandleIdleStateExit()
 {
     // Start watchdog timer to monitor new Query Image session
     StartSelectedTimer(SelectedTimer::kWatchdogTimer);
 }
 
-void DefaultOTARequestorDriver::HandleIdleState(IdleStateReason reason)
+void DefaultOTARequestorDriver::HandleIdleStateEnter(IdleStateReason reason)
```

Comment:
HandleIdleStateEnter() currently starts the Periodic Query timer regardless if there is a Delay timer running or not.  If we want only one of Delay timer, Periodic Query timer, or Watchdog timer to be running at any given time, I was thinking we can make that change in a follow-up PR.